dashboard/app/helpers/survey_results_helper.rb

```diff
@@ -13,6 +13,11 @@ def show_diversity_survey?(kind)
     return false unless has_any_students?
     return false unless has_any_student_under_13?
 
+    diversity_audience = DCDO.get('diversity_audience', 'all')
```

Comment:
Not the expert here, but would it be safer to set the default value of the `DCDO.get` to 'none', then have `return true if diversity_audience == 'all'`? That way, if we forget to set the DCDO flag at all, nothing will happen.

Reply:
The `..., 'all')` argument at the end specifies what should be returned if the flag is unset. Same effect as what you propose.

Reply:
Hmm, isn't it currently the opposite? If the 'diversity_audience' flag is unset, `diversity_audience` will be set to 'all' - and then won't we hit `return true` on line 22?

Reply:
Yes if the flag is unset `diversity_audience` will be `all`, which will not cause any of these three lines looking at `diversity_audience` to return false, so we get to the end and return true, which means the survey will be shown.

Reply:
Ah ok, no difference in how we're thinking through the logic. Just making a note for posterity that we will still need to "turn on" each year's diversity survey via PR (a la #57997) in addition to setting this DCDO `diversity_audience` flag to 'all' (or nil). Thanks Eric!